pkg/skaffold/deploy/label/labeller.go

```diff
@@ -28,7 +28,7 @@ const (
 	RunIDLabel           = "skaffold.dev/run-id"
 )
 
-var runID = uuid.New().String()
+var RunID = uuid.New().String()
```

Comment:
I have to admit to not being a fan of globals like this.  Is there no way to have this on the run context, and have the value be fed into the labeller?

Reply:
seems like this is doable 👍🏼 I'll work on that

Reply:
done